opengrok-web/src/main/java/org/opengrok/web/api/v1/controller/SuggesterController.java

```diff
@@ -98,6 +99,7 @@
     @GET
     @Authorized
     @CorsEnable
+    @Timed
     @Produces(MediaType.APPLICATION_JSON)
     public Result getSuggestions(@Valid @BeanParam final SuggesterQueryData data) throws ParseException {
```

Comment:
This will create metric named `org.opengrok.web.api.v1.controller.SuggesterController.getSuggestions` - not sure we want to expose the full class names in metrics.

Reply:
No longer relevant.